mattermost.py

```diff
@@ -27,7 +27,7 @@
 VERSION = "0.2.0"
```

Comment:
I am sorry, did you mean to increment the version to 0.3.0 as its already at 0.2.0 ?